index.html

```diff
@@ -0,0 +1,286 @@
+    <div class="text">
+      <div class="headline-text">
+          <h1 class="second-text">About Myself</h1>
+
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a
+             project you need coded, don’t hestiate to contact me.</p>
+
+            <a href="#" class="btn-connect"><h6>Let's connect</h6></a>
+          <ul class="medium">
+              <li class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></li>
+              <li class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></li>
+              <li class="social-media"><a href="#"><i class="fa fa-medium"></i></a></li>
+              <li class="social-media"><a href="#"><i class="fa fa-github"></i></a></li>
+              <li class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></li>
+          </ul>
+          <br>
+          <a href="#" class="btn-one">Get My Resume</a>
+          </div>
+       </div>
+
+    <div class="text">
+    <ul class="stacks">
+        <li class="languages">
+            
+            <a href="#"> <h2>Languages</h2></a>
+            <ul class="grid-container">
+            <li>
+                
+                <img src="image/js.png" alt="js_logo">
+                <p>JavaScript</p>
+                
+            </li>
+            <li>
+                
+                <img src="image/html.png" alt="html_logo">
+                <p>HTML</p>
+                
+            </li>
+                
+            <li class="langi">
+              <img src="image/css.png" alt="css_logo">
+                <p>CSS</p>
+            </li>
+        
+          </ul>
+        </li>
+        
+        <li>
+            <a href="#"> <h2>Frameworks</h2> </a>
+        </li>
+        
+        <li>
+            
+          <a href="#"> <h2>Skills</h2> </a>
+  
+        </li>
+    </ul>
+    </div>
+</div>
+</div>
+</section>
+
+<section class="img-b">
+  <img src="../image/headerbackground.png" alt="">
+</section>
+
+<section class="form">
+  <h2></h2>
+  <img src="../image/contactbackground" alt="backgroud-imgae">
+</section>
+
+<section class="ecli">
+  <img src="../image/contactcircle.png" alt="icons">
+</section>
+
+<p class="form-text">Contact me</p>
+<p class="form-paragraph">If you have application you are interested in developing,<br> a feature that you need built or a project
+  that needs coding, <br>i'd love to help with it.
+</p>
+
+<section class="forms">
+<form action="#" method="post">
+  <input type="text" id="fname" name="firstname" maxlength="30" placeholder="Ellis Ng'ona" required><br><br>
+  <input type="email" id="email" name="_replyto" placeholder="ellisallday@gmail.com" required><br><br>
+  <textarea name="messaage" id="subject" maxlength="500" placeholder="write your messaage here" required></textarea>
+  <input type="submit" id="bttn" value="Get in touch">
+</form>
+</section>
```

Comment:
- [ ] Please make sure you are calling your CSS properly, Look at the image below for more details, it is displaying nothing.
![form3](https://user-images.githubusercontent.com/55635977/126405820-96472d38-f736-4896-b648-ae4120f5600d.png)
- [ ] Form uses the action and method attribute to integrate with [formspree](https://formspree.io/html) service.

         * action value should be similar to https://formspree.io/f/{form_id}
         * method value equals POST